src/views/preview/preview.scss

```diff
@@ -1,24 +1,24 @@
 @import "../../colors";
 @import "../../frameless";
 
-html,
-body,
-#app,
+/* override view padding for share banner */
+#view {
+    padding: 0 0 20px 0;
+}
+
 .gui {
+    position: absolute;
+    top: 0;
+    z-index: 11;
```

Comment:
What's the need for the z-index here when the GUI is the only thing being rendered?

Reply:
Probably cruft left over.